lib/clowne/ext/yield_self_then.rb

```diff
@@ -14,11 +14,8 @@ def yield_self
         alias_method :then, :yield_self
       end
 
-      # See https://github.com/jruby/jruby/issues/5220
-      ::Object.include(Ext) if RUBY_PLATFORM.match?(/java/i)
-
       refine Object do
-        include Ext
+        extend Ext
```

Comment:
We should remove this module (and file) completely; and all the references. `Object#then` is available on all supported platforms/versions.

Reply:
Thanks for clarifying that point, would have been useful to have had that guidance earlier but we are where we are.